var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -17,6 +17,7 @@ class Libsonata(CMakePackage):
     git = "https://github.com/BlueBrain/libsonata.git"
 
     version('develop', branch='master', submodules=False, get_full_repo=True)
+    version('0.1.10', submodules=False, get_full_repo=True)
```

Comment:
I don't understand the 'external' from below (like what was external?)

I suspect someone can update this?

Reply:
I don't recall what he issue there is/was, but those versions should go soon™